### PR TITLE
#149 side nav closing

### PR DESCRIPTION
**Pull request by mecskyverse**
# Description

Fixes:  #149 
Side bar Head is overlapping with sidebar body hence it is causing problem to close the Side Navbar. 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] My changes generate no new warnings.

---

## ATTACH SCREEN-SHOTS / DEPLOYMENT LINK:
Before
image
After
image

### CODE REVIEW

**Comment by mecskyverse** [comment]
Hey! @Kanika637 can you review this one.

File: README.md [hunk 1]
```diff
@@ -114,14 +115,14 @@ git checkout -b "Your Branch Name"
...
-// #343 is the issu ticket number and make a request is the assigned issue na
+// #343 is the issu ticket number and make a request is the assigned issue name.
 
 git checkout -b "#343-make a request"
 ```
 
 You are all set!! 🍾🎉. You can now make your changes locally. Once you are done making all your changes, you can now push code and make a pull request.
 
-## Steps to making a pull request  <a name = "raise_pr"></a>
+## Steps to making a pull request <a name = "raise_pr"></a>
```
Comment (Kanika637) [inline README.md:125]:
```suggestion
## Steps for making a pull request <a name = "raise_pr"></a>
```

File: README.md [hunk 2]
```diff
@@ -114,14 +115,14 @@ git checkout -b "Your Branch Name"
 A good branch name would begin with the issue ticket and the assigned issue. See below 👇
 
 ```
-// #343 is the issu ticket number and make a request is the assigned issue na
+// #343 is the issu ticket number and make a request is the assigned issue name.
```
Comment (Kanika637) [inline README.md:118]:
```suggestion
// "#343" is the issue ticket number and "make a request" is the assigned issue name.
```

**Review by Kanika637 — CHANGES_REQUESTED** [review]

**Comment by mecskyverse** [comment]
It's my first time encountering a change request and I have added two commits as per your suggestion. Is it good to go or should I add something else?

**Review by Kanika637 — APPROVED** [review]
LGTM🚀

**Comment by Kanika637** [comment]
> It's my first time encountering a change request and I have added two commits as per your suggestion. Is it good to go or should I add something else?

Yes you did it right👍

**Pull request merged**